### PR TITLE
[BUDS-1901] Document Guide on Budget Changes API

### PR DESCRIPTION
**Pull request by mrcola91**
## Problem
* Transitioning from the Budget Modifications API to the Budget Changes API is not a straightforward process for Integrators

## Solution
* Provide detailed documentation on how the old and new APIs function in relation to each other.

## High-Level Changes
* Adds a tutorial document for understanding the Budget Changes API
* Adds navigation to new document from Project Level resources page
* Alphabetizes the Tutorials sidebar

[Understanding the Budget Changes API.pdf](https://github.com/procore/documentation/files/9984520/Understanding.the.Budget.Changes.API.pdf)

### CODE REVIEW

**Review by zbluewave — APPROVED** [review]
Not familar with the code style in this repo. But the content looks good to me

File: tutorials/tutorial_budget_changes_api.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+| | The WBS Code of the `From` Budget Line Item is used to set the `wbs_code_id` of the Adjustment Line Item when migrating from Budget Modifications to Budget Changes | | |
+| `to_budget_line_item_id` | ID of the Budget Line Item from which the Budget Modification is being credited | `wbs_code` | This attribute on Adjustment Line Items with type `change_event` indicates the WBS Code for which the Adjustment is being credited |
+| | The WBS Code of the `To` Budget Line Item is used to set the wbs_code_id of the Adjustment Line Item when migrating from Budget Modifications to Budget Changes | | |
+| `notes` | Field for entering information about a Budget Modification | `description` | Field for entering a custom description of an adjustment |
+| | | `comment` | Field for entering a comment about an adjustment |
+| `transfer_amount` | Amount being transferred from one Budget Line Item to another | `amount` | Amount being withdrawn or credited or the Adjustment Line Item |
```
Comment (donwong2) [inline tutorials/tutorial_budget_changes_api.md:93]: In the last column `Amount being withdrawn or credited or the Adjustment Line Item`- I am not sure if the last part `or the Adjustment Line Item` should be `for the Adjustment Line Item`, or we should use a comma somewhere?  (I ain't good at english 😭 )

File: tutorials/tutorial_budget_changes_api.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+
+Similarly to the [Create Budget Modification](https://developers.procore.com/reference/rest/v1/budget-modifications?version=1.0#create-budget-modification) endpoint, the [Update Budget Modification](https://developers.procore.com/reference/rest/v1/budget-modifications?version=1.0#update-budget-modification) endpoint also will return a `405 - Method not Allowed` response with an `Allow` header describing that `GET` is the only allowed method.
+
+The [Update information of a Budget Change](https://developers.procore.com/reference/rest/v1/budget-change?version=1.0#update-information-of-a-budget-change) endpoint behaves similarly to the [Create a Budget Change](https://developers.procore.com/reference/rest/v1/budget-change?version=1.0#create-a-budget-change) endpoint, where the request body is almost exactly the same with the exception of an `id` attribute for the Adjustment Line Items in a Budget Change, as well as an `id` attribute alongside other `Budget Change attributes like `title` and `status`. All fields are not required for updates, only `id` and the attribute or attributes that need to be updated.
```
Comment (donwong2) [inline tutorials/tutorial_budget_changes_api.md:190]:
```suggestion
The [Update information of a Budget Change](https://developers.procore.com/reference/rest/v1/budget-change?version=1.0#update-information-of-a-budget-change) endpoint behaves similarly to the [Create a Budget Change](https://developers.procore.com/reference/rest/v1/budget-change?version=1.0#create-a-budget-change) endpoint, where the request body is almost exactly the same with the exception of an `id` attribute for the Adjustment Line Items in a Budget Change, as well as an `id` attribute alongside other `Budget Change` attributes like `title` and `status`. All fields are not required for updates, only `id` and the attribute or attributes that need to be updated.
```

I think a single quote was missed.

**Review by donwong2 — COMMENTED** [review]
Very small things I think

**Review by jimj-procoretech — APPROVED** [review]

**Review by jwong0 — APPROVED** [review]

**Pull request merged**